### DISCUSSION TITLE
lock component position in layer manager

### DISCUSSION THREAD

**Discussion by badiuciprian** [GrapesJS/grapesjs#5096] [Q&A]
Hi to all, I have a project where I need an image as background. All good, but when adding another component on the canvas it gets placed behind the image. Is there a way to lock the image placement to be always behind the other components? Thanks

**Comment by artf** [comment]
Please provide a reproducible demo of your use case.

**Reply by badiuciprian** [reply]
Hi, unfortunately I cannot provide an example but I attached an image that exemplifies my question. I'm searching for a way to lock the position of the image in the layer manager, so that it's zIndex remains the same. 
  
![sfsrvr](https://user-images.githubusercontent.com/82867789/236811886-6789b3e3-2a91-4f63-ac55-821202172c79.png)
